go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/NYTimes/gziphandler v1.1.1
 	github.com/alicebob/miniredis/v2 v2.30.2
 	github.com/aquasecurity/bolt-fixtures v0.0.0-20200903104109-d34e7f983986
-	github.com/aquasecurity/defsec v0.88.1
+	github.com/aquasecurity/defsec v0.88.2-0.20230516215146-673ff3afe374
```

Comment:
note: update to latest when https://github.com/aquasecurity/trivy/pull/4474 is merged